doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -304,6 +304,38 @@ Values are provided in kilobytes (KB).
       | :ref:`Window Covering <matter_window_covering_sample>` (Release)   |            52 |               568 |              4 |         48 |         672 |                             158 |
       +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
 
+   .. tab:: nRF54LM20 DK with nRF7002-EB2
```

Comment:
```suggestion
   .. tab:: nRF54LM20 DK with nRF7002-EB II
```
My bad, missed in the initial review. https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf70/index.html